src/api/layers/GeoAdminGroupOfLayers.class.js

```diff
@@ -22,7 +22,7 @@ export default class GeoAdminGroupOfLayers extends GeoAdminLayer {
      * @param {GeoAdminLayer[]} layers Description of the layers being part of this group
      */
     constructor(id, name, layers) {
```

Comment:
I would also use an object here a parameter to be consistent with other AbstractLayers